scripts/automation/Radius/Functions/Private/menus/Show-CertDeploymentMenu.ps1

```diff
@@ -0,0 +1,17 @@
+function Show-CertDeploymentMenu {
+    $title = ' JumpCloud Radius Cert Deployment '
+    Clear-Host
+    Write-Host $(PadCenter -string $Title -char '=')
+    Write-Host $(PadCenter -string "Select an option below to view certificate deployment status`n" -char ' ') -ForegroundColor Yellow
+
+    # ==== instructions ====
+    Write-Host $(PadCenter -string ' Certificate Deployment Result Options ' -char '-')
+    # List options:
+    Write-Host "1: Press '1' to view results."
```

Comment:
Remove period for consistency with other items

Reply:
resolved in ae09d4e22dbe28c6fd5acf2a5ef960af3fbae2ab, I went with adding a period because most menus were written this way.